docs/docs/tutorials/adding_annotations.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: Adding the grpc-gateway annotations to an existing protobuf file
+parent: Tutorials
+nav_order: 6
+---
+
+## Adding the grpc-gateway annotations to an existing protobuf file
+
+Now that we've got a working Go gRPC server, we need to add the grpc-gateway annotations:
+
+```proto
+syntax = "proto3";
+
+package helloworld;
+
+import "google/api/annotations.proto";
+
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {
+        option (google.api.http) = {
+        post: "/v1/example/echo"
+        body: "*"
+    };
+  }
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+### Generating the grpc-gateway stubs
+
+Now that we've got the grpc-gateway annotations added to the proto file, we need to use the grpc-gateway generator to generate the stubs.
+
+Before we can do that, we need to copy some dependencies into our protofile structure. Copy the `third_party/googleapis` folder from the grpc-gateway repository to your local protofile structure. It should look like this afterwards:
+
+''' (backticks)
+proto/
+helloworld/
+hello_world.proto
+google/
+api/
+http.proto
+annotations.proto
+''' (backticks)
+
+#### Using buf
+
+[Using buf](generating_stubs/using_buf.md)
```

Comment:
This isn't the right place for these links, we need to talk about what needs to be modified in these workflows for the generation of grpc-gateway stubs. In the case of buf:
```suggestion
We'll need to add the grpc-gateway generator to the generation configuration:
''' (backticks)
plugins:
   ... other plugins...
  - name: grpc-gateway
    out: proto
    opt: paths=source_relative
''' (backticks)

And that's it! Now if you run:

''' (backticks)
$ buf generate
'''

It should produce a `*.gw.pb.go` file.
```